gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -240,7 +266,7 @@ public Builder toBuilder() {
   }
 
   static ZoneInfo fromPb(com.google.api.services.dns.model.ManagedZone pb) {
-    Builder builder = new Builder(pb.getName());
+    Builder builder = new BuilderImpl(pb.getName());
```

Comment:
I just realized that `fromPb` and `toPb` are not symmetrical in respect to nameServers.
I also think that a better API will return an empty list (and not null), however code could be change
to fix fromPb/toPb